lib/galaxy/config/schema.py

```diff
@@ -52,8 +52,11 @@ def _preprocess(self, app_schema):
         self._reloadable_options = set()  # config options we can reload at runtime
         self._paths_to_resolve = {}  # {config option: referenced config option}
         self._per_host_options = set()  # config options that can be set using a per_host config parameter
+        self._deprecated_aliases = {}
```

Comment:
```suggestion
        self._deprecated_aliases = {}  # {old key: new key}
```
Maybe add this comment to avoid confusion, since in the for loop below it does the opposite of what `_paths_to_resolve` does? (and it looked like a bug when I first saw it - but it could be just me..)